build.sbt

```diff
@@ -189,10 +189,10 @@ lazy val V = new {
   val scala211 = "2.11.12"
   val sbtScala = "2.12.12"
   val scala212 = "2.12.13"
-  val scala213 = "2.13.5"
+  val scala213 = "2.13.6"
   val ammonite212Version = scala212
-  val ammonite213Version = scala213
-  val scalameta = "4.4.17"
+  val ammonite213Version = "2.13.5"
+  val scalameta = "4.4.17+11-d05165cd-SNAPSHOT"
```

Comment:
Running the release here: https://github.com/scalameta/scalameta/actions/runs/850497093

Wanted to test it out first.